starcluster/awsutils.py

```diff
@@ -772,8 +773,27 @@ def get_all_instances(self, instance_ids=None, filters=None):
                 return []  # Haven't created the security group in aws yet
             del filters['group-name']
 
-        reservations = self.conn.get_all_instances(instance_ids,
-                                                   filters=filters)
+        for i in xrange(5):
+            try:
+                reservations = self.conn.get_all_instances(instance_ids,
+                                                           filters=filters)
+                break
+            except EC2ResponseError as e:
+                if instance_ids:
+                    log.error("instance_ids filtering based on: "
```

Comment:
That's fatal? (it is not going to work even if we retry?)
Would it be worth it to log the actual exception text here?